### PR TITLE
reorder folders and added config to run script in vue

### PR DESCRIPTION
**Pull request by LucasZibell**
## Summary

- Change eslint logic to measure lint errors
- Reorder files to have a better distribution of vue/react/angular metrics
- Added basic configuration to run script on a vue project
- Measuring 8 metrics from a Vue project

### CODE REVIEW

File: src/buildChecks/constants.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
 const { createObject } = require('../utils');
 
-module.exports = createObject(['BUILD_TIME', 'APP_SIZE']);
+module.exports.buildMetrics = createObject(['BUILD_TIME', 'APP_SIZE']);
```
Comment (FrankIglesias) [inline src/buildChecks/constants.js]: define a way to inject these variables somehow

File: src/frontEnd/reactChecks/constants.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
 const { createObject } = require('../../utils');
 
-module.exports = createObject(['REDUX_RECOMPOSE']);
+module.exports.reactMetrics = createObject(['REDUX_RECOMPOSE', 'INDEX_LINES', 'LAYOUT_LINES', 'I18N']);
+
+module.exports.limits = {
+  lines: 150
```
Comment (FrankIglesias) [inline src/frontEnd/reactChecks/constants.js]: 🤔  check the new eslint values for react

File: src/frontEnd/vueChecks/createSummary.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = reports => {
+  const summary = [];
+  return [...summary, ...reports];
```
Comment (FrankIglesias) [inline src/frontEnd/vueChecks/createSummary.js]: missing TODO?

File: src/buildChecks/constants.js (hunk 1 above)
Comment (LucasZibell) [reply src/buildChecks/constants.js]: Yup, we are going to define two new args to define the buildPath and the extraBuildParams. @alejobh Is going to apply it in the next PR

File: src/frontEnd/reactChecks/constants.js [hunk 4]
```diff
@@ -1,3 +1,7 @@
 const { createObject } = require('../../utils');
 
-module.exports = createObject(['REDUX_RECOMPOSE']);
+module.exports.reactMetrics = createObject(['REDUX_RECOMPOSE', 'INDEX_LINES', 'LAYOUT_LINES', 'I18N']);
```
Comment (alejobh) [inline src/frontEnd/reactChecks/constants.js]: many projects are not using `REDUX_RECOMPOSE` instead we are using Context with globalReducer, and reducers for screens, what about this for the future?

File: src/buildChecks/constants.js [hunk 5]
```diff
@@ -1,3 +1,13 @@
...
+
+module.exports.buildPath = {
+  react: 'build',
+  vue: 'dist'
+};
+
+module.exports.extraBuildParams = {
+  react: 'development',
+  vue: ''
+};
```
[the remark this answers is not in the document]
Comment (LucasZibell) [reply src/buildChecks/constants.js]: Nice change here. I think we are going to delete this and send the buildPath and extraBuildParams with parameters

File: src/frontEnd/reactChecks/constants.js (hunk 4 above)
Comment (LucasZibell) [reply src/frontEnd/reactChecks/constants.js]: Yup you are 100% right. We should design a way ('if necessary') to measure context and global reducers. Above that maybe we can have the two metrics and apply each one to a specific project. What do you think?

File: src/buildChecks/constants.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (alejobh) [reply src/buildChecks/constants.js]: oukey! i assume that i will make this change haha

File: src/frontEnd/reactChecks/constants.js (hunk 4 above)
Comment (alejobh) [reply src/frontEnd/reactChecks/constants.js]: yeap, agree 100%.
But those metrics can be optional, just one could be necessary. for example: if the projects uses redux_recompose its ok, if uses context its ok, but if uses none: bad metric. What do you think?

**Pull request merged**